libraries/AC_AttitudeControl/AC_AttitudeControl_TS.cpp

```diff
@@ -55,18 +55,28 @@ void AC_AttitudeControl_TS::relax_attitude_controllers(bool exclude_pitch)
     }
 }
 
-// Command euler yaw rate and pitch angle with roll angle specified in body frame
-// (used only by tailsitter quadplanes)
-// If plane_controls is true, swap the effects of roll and yaw as euler pitch approaches 90 degrees
-void AC_AttitudeControl_TS::input_euler_rate_yaw_euler_angle_pitch_bf_roll(bool plane_controls, float body_roll_cd, float euler_pitch_cd, float euler_yaw_rate_cds)
+// Commands a body-frame roll angle (in centidegrees), an euler pitch angle (in centidegrees), and a yaw rate (in centidegrees/s).
+// See input_euler_rate_yaw_euler_angle_pitch_bf_roll_rad() for full details.
+void AC_AttitudeControl_TS::input_euler_rate_yaw_euler_angle_pitch_bf_roll_cd(bool plane_controls, float body_roll_cd, float euler_pitch_cd, float euler_yaw_rate_cds)
 {
     // Convert from centidegrees on public interface to radians
-    float euler_yaw_rate = cd_to_rad(euler_yaw_rate_cds);
-    float euler_pitch    = cd_to_rad(constrain_float(euler_pitch_cd, -9000.0f, 9000.0f));
-    float body_roll      = cd_to_rad(-body_roll_cd);
+    float euler_yaw_rate_rads = cd_to_rad(euler_yaw_rate_cds);
+    float euler_pitch_rad    = cd_to_rad(euler_pitch_cd);
+    float body_roll_rad      = cd_to_rad(-body_roll_cd);
```

Comment:
```suggestion
    float body_roll_rad      = cd_to_rad(body_roll_cd);
```
The negative is done on line 76 in `input_euler_rate_yaw_euler_angle_pitch_bf_roll_rad`